src/id3_variants_training/__main__.py

```diff
@@ -9,12 +11,16 @@ def main():
                         help='use remote API to access variant information rather than local VCF files')
     args = parser.parse_args()
 
-    # creates ConfusionMatrix object with a path to the config file and whether to use local VCF files or server VCF files
     use_local_vcf_files = not args.use_candig_apis
-    conf_matrix = ConfusionMatrix(getattr(args, 'config-file'), use_local_vcf_files)
+    config_file_path = getattr(args, 'config-file')
+
+    # get either a LOCAL_API or GA4GH_API object depending on the command line arguments passed in
+    api = get_api(config_file_path, use_local_vcf_files, True)
+
+    conf_matrix = ConfusionMatrix(api)
```

Comment:
So I think what we want to do for train-id3 is just train the ID3 model, then save it.  I think generating the predictions from the ID3 tree given some local data, and then generating a confusion matrix from the local data and the predictions, could usefully be separate scripts (that, amongst other things, let us test our trained model against multiple datasets).   This would also allow a second, larger refactoring of the ConfusionMatrix code.